helm-charts/explorer/templates/explorer-deployment.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.explorer.name }}
+  labels:
+    name: {{ $.Values.explorer.name }}
+    app.kubernetes.io/name: {{ $.Values.explorer.name }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+  annotations: 
+    {{- toYaml .Values.annotations | nindent 6 }}      
+
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ $.Values.explorer.name }}
+      app.kubernetes.io/name: {{ $.Values.explorer.name }}
+      helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ $.Values.explorer.name }}
+        app.kubernetes.io/name: {{ $.Values.explorer.name }}
+        helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+    spec:
+      volumes:
+        - name: explorer-config
+          configMap:
+            name: explorer-config
+            items:
+              - key: config.json
+                path: config.json
+              - key: basic-network.json
+                path: basic-network.json
+
```

Comment:
[nitpick]: better naming can be given to the file.

Reply:
name has been changed to test-network.json